build-go.sh

```diff
@@ -50,14 +50,18 @@ function doBuild() {
 	local goarch=$2
 	local target=$3
 	local output=$4
+  local version=$5
+
+  dir=$output
+	name=$(echo $target | awk -F'/' '{ print $3 }')
+  binname=${name}_${version}_${goos}-${goarch}
 
 	echo "==> building for $goos $goarch"
 
-	dir=$output/$1-$2
-	if [ -e $dir ]; then
-		echo "    $dir exists, skipping build"
-		return
-	fi
+	# if [ -e $dir ]; then
+	# 	echo "    $dir exists, skipping build"
```

Comment:
I don't think we should force it, but the issue is that this needs to be either moved so that we only check it once per version (not per build as at the moment)  or we need to change the check to check folder + target file.